_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -244,13 +251,17 @@ The `s3` source includes the following metrics:
 * `s3ObjectsNotFound`: The number of S3 objects that the `s3` source failed to read due to an S3 "Not Found" error. These are also counted toward `s3ObjectsFailed`.
 * `s3ObjectsAccessDenied`: The number of S3 objects that the `s3` source failed to read due to an "Access Denied" or "Forbidden" error. These are also counted toward `s3ObjectsFailed`. 
 * `s3ObjectsSucceeded`: The number of S3 objects that the `s3` source successfully read.
+* `s3ObjectNoRecordsFound`: The number of S3 objects that resulted in 0 records added to the buffer by the `s3` source.
+* `s3ObjectsDeleted`: The number of S3 objects deleted by the `s3` source.
+* `s3ObjectsDeleteFailed`: The number of S3 objects that the `s3` source failed to delete.
+* `s3ObjectsEmpty`: The number of S3 objects that are considered empty since they have a size of `0`. These objects will be skipped by the `s3` source.
```

Comment:
```suggestion
* `s3ObjectsEmpty`: The number of S3 objects that are considered empty because they have a size of `0`. These objects will be skipped by the `s3` source.
```